dotcom-rendering/src/components/ChartAtom.stories.tsx

```diff
@@ -38,15 +38,15 @@ const Wrapper = ({ children }: { children: React.ReactNode }) => (
 export const BarChart = () => {
 	return (
 		<Wrapper>
-			<ChartAtom id="123abc" html={barChartHtml} />
```

Comment:
This is unrelated to my PR but I rebased main and main is failing tsc validation because of the missing prop.